WalletWasabi.Fluent/Models/Wallets/TransactionModel.cs

```diff
@@ -40,7 +40,7 @@ public partial class TransactionModel : ReactiveObject
 
 	public Money? Fee { get; init; }
 
-	public Money Amount => Math.Abs(IncomingAmount ?? OutgoingAmount ?? Money.Zero);
+	public Money Amount => IncomingAmount ?? -(OutgoingAmount ?? Money.Zero);
```

Comment:
I believe the amount should be either negative or positive, depending if it was an outgoing or incoming transaction.

Previously it was always positive which was misleading.

Reply:
WDYM? What is this amount field?

Reply:
@yahiheb the `TransactionModel.Amount` property is a unification of the `IncomingAmount` and `OutgoingAmount` (as shown above) and is used in several places in the UI logic that creates the transaction tree (`TransactionTreeBuilder`). In the future if these columns are unified in the UI the `Amount` value can be displayed in the UI as well.